middlewares/selectAndAuthzInstructorQuestion.sql

```diff
@@ -21,7 +21,9 @@ FROM
   issue_count
 WHERE
   q.id = $question_id
-  AND q.course_id = $course_id
+  AND q.course_id = $course_id -- I'm not really sure what to do here. This authorization is used for TONS of stuff.
```

Comment:
Agreed. I think we keep this for now and add an `OR (however we determine question sharing access)` when we implement sharing.

Reply:
I will add an explicit TODO comment here that can be merged here, then changed in the main question sharing PR.